Formula/llvm.rb

```diff
@@ -225,19 +282,19 @@ def install
   end
 
   def caveats
-    s = <<-EOS.undent
+    caveat_message = <<-EOS.undent
```

Comment:
It may be bad practice, but there's no other string in this method, so the meaning is obvious from context and changing it now would lead to inconsistency across formulae, which is also bad.